controllers/drpolicy_controller.go

```diff
@@ -382,6 +382,16 @@ func (r *DRPolicyReconciler) configMapMapFunc(configMap client.Object) []reconci
 		return []reconcile.Request{}
 	}
 
+	labelAdded := util.AddLabel(configMap, util.OCMBackupLabelKey, util.OCMBackupLabelValue)
```

Comment:
I would move this to [ConfigMapGet](https://github.com/RamenDR/ramen/blob/dbb1fe057731b32095a642ff8891297d559e8589/controllers/ramenconfig.go#L207) function instead.

This routine is called on every config map update as part of the reconciler watch. Instead, moving this to the routine above helps add the label across the code base where the config map is accessed, on the hub as well as on the managed cluster (although on the managed cluster there is no backup initiated, so maybe we can latch the label addition out).

Reply:
This is fine. Even if we do not remove ConfigMapGet, the label check is on the object passed in and also the predicate is a resource version change predicate, and would only occur if this was changed. As a result calling this often would not occur.